src/itdelatrisu/opsu/Options.java

```diff
@@ -1065,12 +1070,16 @@ public static File getBeatmapDir() {
 			return beatmapDir;
 
 		// search for directory
-		for (int i = 0; i < BEATMAP_DIRS.length; i++) {
-			beatmapDir = new File(BEATMAP_DIRS[i]);
+		for (String BEATMAP_DIR : BEATMAP_DIRS) {
+			beatmapDir = new File(BEATMAP_DIR);
 			if (beatmapDir.isDirectory())
 				return beatmapDir;
 		}
-		beatmapDir.mkdir();  // none found, create new directory
+		// none found, create new directory
+		assert beatmapDir != null;
```

Comment:
I'd rather not include asserts.  If something goes wrong, let an error get thrown.